website/static/js/old-pages/index.js

```diff
@@ -0,0 +1,264 @@
+          </Card.Footer>
+        </Card>
+      </Col>
+    );
+
+    const Hotspots = () => (
+      <Row>
+        <HotspotCard
+          title={<translate>Runtime Development</translate>}
+          text={
+            <translate>
+              Let us teach you to build a custom blockchain using Substrate.
+            </translate>
+          }
+        >
+          <Button
+            variant='secondary'
+            href={pageUrl('who/runtime-developer')}
+            className='m-1 primary-color'
+          >
+            <translate>Get Started on Substrate</translate>
+          </Button>
+        </HotspotCard>
+        <HotspotCard
+          title={<translate>Front-End Development</translate>}
+          text={
+            <translate>
+              Let us teach you to build interactive user experiences with
+              Polkadot-JS.
+            </translate>
+          }
+        >
+          <Button
+            variant='secondary'
+            href={pageUrl('who/front-end-developer')}
+            className='m-1 primary-color'
+          >
+            <translate>Get Started on Polkadot-JS</translate>
+          </Button>
+        </HotspotCard>
+        <HotspotCard
+          title={<translate>Smart Contract Development</translate>}
+          text={
+            <translate>
+              Let us teach you to build Wasm smart contracts with ink!.
+            </translate>
+          }
+        >
+          <Button
+            variant='secondary'
+            href={pageUrl('who/contract-developer')}
+            className='m-1 primary-color'
+          >
+            <translate>Get Started on ink!</translate>
+          </Button>
+        </HotspotCard>
+      </Row>
+    );
+
+    const WhatIsSubstrate = () => (
+      <div
+        className='productShowcaseSection mb-5 mt-5'
+        style={{ textAlign: 'center' }}
+      >
+        <h2 className='text-dark'>
+          <translate>What is Substrate?</translate>
+        </h2>
+        <h2 className='primary-color-text'>&#11015;</h2>
+      </div>
+    );
+
+    const Substrate1 = () => (
+      <Feature
+        image={`${baseUrl}img/undraw_programming.svg`}
+        imageAlign='right'
+        title={<translate>Everything you Need to Build a Blockchain</translate>}
+        background='light'
+      >
+        <ul className='lead'>
+          <li>
+            <translate>Fast and efficient database.</translate>
+          </li>
+          <li>
+            <translate>Modular P2P networking stack in</translate>{' '}
+            <a href='https://github.com/libp2p'>libp2p</a>
+            <translate>.</translate>
+          </li>
+          <li>
+            <translate>Hot-swappable consensus layer.</translate>
+          </li>
+          <li>
+            <translate>
+              Customizable transaction queue management system.
+            </translate>
+          </li>
+          <li>
+            <translate>Diverse library of runtime modules.</translate>
```

Comment:
```suggestion
            <translate>Rich framework for composing runtime logic.</translate>
```

The term "runtime module" has been replaced with "pallet". I didn't suggest a direct translation because I feared people reading that far down the landing page wouldn't know what a pallet was yet.

Reply:
We may also want to avoid direct translations of pallet as it's also a nod to "crates" and that wordplay probably doesn't translate well.

Reply:
hey there! i noticed that change was suggested to the old home page. I went ahead and added it to the new homepage and also removed the blast banner as per the suggestion from @riusricardo. 

Also, just FYI, I haven't added the strings for translation just yet. That will be in in a future pull request. @ericmorel suggested it was cool to just publish the english version so the new page could be up sooner.

I'm making a pull request now for the changes listed above, and will make another pull request shortly for the new seminar page :) 

Let me know if you guys think of any other changes! Thanks!